gef.py

```diff
@@ -5429,7 +5429,7 @@ def __init__(self) -> None:
         return
 
     @parse_arguments({"type": "", "address": ""}, {})
-    def do_invoke(self, *args: Any, **kwargs: Dict[str, Any]) -> None:
+    def do_invoke(self, _: Any, **kwargs: Dict[str, Any]) -> None:
```

Comment:
this might be broken now, since it's not variadic anymore.

Reply:
oh, i think you're right. sorry for suggesting this. the correct version should be `*_`

Reply:
```suggestion
    def do_invoke(self, *_: Any, **kwargs: Dict[str, Any]) -> None:
```